spec/latest/core/index.html

```diff
@@ -1287,6 +1287,51 @@ <h3>Client initiated pagination</h3>
       to provide a custom page identifier (i.e. a GUID or a letter)
       instead of a number.</p>
   </section>
+
+  <section>
+    <h3>Extensions</h3>
+    <p>While Hydra Core Vocabulary allows to address many usage scenarios,
+      not every aspect of API behavior can be covered with it. This
+      applies especially to querying, resource projection or data structure
+      description. This is due to fact that Hydra was meant to be as light
+      as possible forcing to drop some of the features out of its scope.</p>
+
+    <p>That is why there is a possibility of hinting a client on what kind
+      of extensions may be found or used in the received payloads.
+      After discovering a <i>extension</i> predicate in the API documentation,
+      client can assume additional details are available described with
+      additional vocabularies.</p>
+
+    <p>It is up to the used vocabulary how these additional details should be
+      interpreted and in case client does not recognize these extensions,
+      additional details should be ignored and base Hydra interpretation
+      should be in force.</p>
+
+    <p>Server SHOULD NOT use extensions to add statements that are in
+      contradiction to base Hydra interpretation so the client is not confused.
+      Server SHOULD also keep multiple extensions describing adequate
+      knowledge in line regarding their description (i.e. data structure
+      descriptions in various vocabularies should not cause differences).</p>
+
+    <p>Client can express its preferences through the <i>PREFER</i> header
+      by pointing the preferred extensions via IRIs as on the example below.
+      The expectation of <i>hydra.extension</i> should be followed by a
```

Comment:
You could maybe just refer to the ABNF definition providing by the RFC instead of trying to describe the syntax in plain English? The example should be enough for most readers:

```suggestion
    The client SHOULD use the `Prefer` HTTP header [[!RFC7240]] with the `hydra.extension` preference an an <i>iri</i> attribute having the IRI of the extension as value to hint the server about the extensions it supports.
```